src/Microsoft.Identity.Web/MicrosoftIdentityOptions.cs

```diff
@@ -25,6 +25,11 @@ public class MicrosoftIdentityOptions : OpenIdConnectOptions
         /// </summary>
         public string Domain { get; set; }
 
+        /// <summary>
+        /// Gets or sets the RedirectUri of the Azure Active Directory or B2C tenant, e.g. contoso.onmicrosoft.com.
```

Comment:
```suggestion
        /// In a Web app, gets or sets the RedirectUri of (URI where the token will be sent back by
        /// Azure Active Directory or Azure Active Directory B2C.
        /// This property is exclusive with <see cref="CallbackPath"/> which should be used preferably if you don't want
        /// to have a different deployed configuration from your developper configuration.
        /// There are cases where RedirectUri is needed, for instance when you use a reverse proxy that transforms https
        /// URLs (external world) to http URLs (inside the protected area). This can also be useful for Web apps running
        /// in containers (for the same reasons)
```